client/web/views/root_view/connection_confirm_dialog/index.ts

```diff
@@ -0,0 +1,210 @@
+/*
```

Comment:
Shall we use `//` instead of `/*`? Since all other TypeScript code are using `//`.